common/oauthTokenManager.go

```diff
@@ -318,7 +339,8 @@ func certLoginNoUOTM(tenantID, activeDirectoryEndpoint, certPath, certPass, appl
 		}
 
 		if pk == nil || cert == nil {
-			return nil, fmt.Errorf("could not find the required information (private key) in the supplied .pem file")
+			fmt.Println(pk == nil, cert == nil)
```

Comment:
This looks like debug code. To be removed now, right?

Reply:
Ah, yes, that is debug code.